pyproject.toml

```diff
@@ -51,7 +51,8 @@ pyink-use-majority-quotes = true
 requires = [
     # We build against the most recent supported NumPy 2.0 release;
     # see https://github.com/numpy/numpy/issues/27265
-    "numpy~=2.0",
+    "numpy~=2.0; python_version<'3.13'",
+    "numpy~=2.1; python_version>='3.13'",
```

Comment:
This is fine, but I believe `~=2.0` means "the most recent available version matching `2.*`", so I think this should be fine as-is.

Reply:
Hmm. No idea why, but it wasn't working for me with a 3.13t build. Seems to work now ,though, no idea why.